app-frontend/src/app/components/colorCorrectAdjust/colorCorrectAdjust.controller.js

```diff
@@ -9,13 +9,15 @@ export default class ColorCorrectAdjustController {
             ceil: 2,
             step: 0.1,
             precision: 1,
+            showTicks: 0.25,
             onEnd: (id, val) => this.onFilterChange(id, val)
         };
 
         let baseFilterOptions = {
-            floor: 0,
+            floor: -60,
```

Comment:
as far as I can tell, negative parameters are supported, we just didn't let the user specify them.